clang/test/dpct/cache_remove.cu

```diff
@@ -0,0 +1,50 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none -out-root %T/cache_remove %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/cache_remove/cache_remove.dp.cpp --match-full-lines %s
+#include <cuda_runtime.h>
+
+__global__ void kernel()
+{
+  return;
+}
+
+int main()
+{
+  cudaStream_t stream;
+  const int pre_transform_tensor_size =
+      8 * 8 * 8;
+  const int transformed_tensor_size = pre_transform_tensor_size * 36 / 16;
+  const int res_block_mem =
+      transformed_tensor_size * 2 + pre_transform_tensor_size;
+  float *aptr;
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaLaunchAttributeValue is not supported.
+  cudaStreamAttrValue stream_attribute = {};
+  stream_attribute.accessPolicyWindow.base_ptr = aptr;
+  stream_attribute.accessPolicyWindow.num_bytes = res_block_mem;
+  stream_attribute.accessPolicyWindow.hitRatio = 1.0f;
+  stream_attribute.accessPolicyWindow.hitProp = cudaAccessPropertyPersisting;
+  stream_attribute.accessPolicyWindow.missProp = cudaAccessPropertyStreaming;
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaStreamSetAttribute is not supported.
```

Comment:
can remove all the cudaStreamSetAttribute(..) here?

Reply:
> can remove all the cudaStreamSetAttribute(..) here?

no, cudaStreamSetAttribute be used for setting stream attribute include cudaLaunchAttributeAccessPolicyWindow for cache, Synchronization Policy, Priority.
So, we need check the 2nd param value of cudaStreamSetAttribute to determine whether the call is used to setting cache config.

Only this call be used to setting cudaStream cache config, we will remove this call.